Doyeon-k-fanshawe/group2_bootcamp
# Adding Promo Video

## What does the PR do?
* It adds the promo video to the assets folder

## How do you check this PR?
* Pull down the branch
*  See added video

## Screenshots 
None.
